doc/qat/USAGE.md

```diff
@@ -10,7 +10,7 @@
 
 Deploy DeviceClass, Namespace and ResourceDriver
 ```bash
-kubectl apply -f deployments/qat/device-class.yaml
+kubectl apply -f deployments/qat/resource-class.yaml
```

Comment:
`device-class.yaml` is correct, the git tree had old name of the file, see #23

Reply:
Thank you! :)